### DISCUSSION TITLE
3.5, 5, 7inch LCD

### DISCUSSION THREAD

**Discussion by homepods** [Arksine/moonraker#50] [Feature Requests]
I would like to create a linux app such as 'octodash' to run moonkraker with klipper.

Point me in the right direction. How is PanelDue functioning? PanelDue has built in controller?
How can we make any LCD work?

**Comment by jordanruthe** [comment]
Check out KlipperScreen: https://github.com/jordanruthe/KlipperScreen

**Comment by Arksine** [comment]
@jordanruthe Nice work!  Would you like me to add a link to KlipperScreen to Moonraker's readme?

@homepods The PanelDue is a unique display, it communicates with Moonraker over a serial connection.  Moonraker supplies it with printer status in json format and receives commands in the form of gcode.   Jordan's KlipperScreen is definitely what you are after.  Its using Moonraker's remote API, which generally speaking is available via both HTTP and Websocket.  The advantage to this approach is that KlipperScreen is not only capable of running on the same Pi as Moonraker, it is also capable of running on a standalone Pi and connecting over the network.

**Reply by jordanruthe** [reply]
That would be great if you could add a link in the readme to KlipperScreen.
